internal/pkg/instrumentation/probe/sampling/sampling.go

```diff
@@ -0,0 +1,123 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package sampling
+
+import "github.com/cilium/ebpf"
+
+type SamplerType uint64
+
+// SamplerType defines the type of a sampler.
+const (
+	// OpenTelemetry spec-defined samplers.
+
+	SamplerAlwaysOn SamplerType = iota
+	SamplerAlwaysOff
+	SamplerTraceIDRatio
+	SamplerParentBased
+
+	// Custom samplers TODO.
+)
+
+// Manager is used to configure the samplers used by eBPF.
+type Manager struct {
+	// TODO
+}
```

Comment:
This is not used in this PR. Can you remove it, `NewSamplingManager`, and the use of `NewSamplingManager` from this change? In the place where `NewSamplingManager` is called in the base probe, can you leave a comment explaining how sampling is planned to be setup and used instead?

Reply:
Done